test-projects/social-login/setupAuthRoutes.js

```diff
@@ -0,0 +1,98 @@
+const { startAuthedSession } = require('@keystone-alpha/session');
+
+module.exports = ({
+  strategy,
```

Comment:
This function should take a `server` parameter, since the strategy itself will no longer have a `server` config option.